src/python/grpcio/grpc/__init__.py

```diff
@@ -23,6 +23,11 @@
 
 logging.getLogger(__name__).addHandler(logging.NullHandler())
 
+try:
+    from _grpcio_metadata import __version__
```

Comment:
Do we know where this is being set?

Reply:
`_grpcio_metadata` is generated from a template located [here](https://github.com/grpc/grpc/blob/4d5c3102a1b1bd440f22d4889e3afd146997fb6d/templates/src/python/grpcio/grpc/_grpcio_metadata.py.template). It looks like all of the templates in the `templates/` directory are being rendered by a script located [here](https://github.com/grpc/grpc/blob/806c1e0b7abcde983827db1ef1dd6a3785f0a5c8/tools/buildgen/generate_projects.sh).